jjb/clm/clm_go.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- project:
+    name: clm-go-project-view
+    project-name: clm-go
+    views:
+      - project-view
+
+- project:
+    name: go-clm
+    jobs:
+      - gerrit-go-clm
+    nexus-iq-namespace: "magma-"
+    project: "magma/magma"
+    project-name: "clm-go"
```

Comment:
Ideally, "project-name" should contain the name of the repo so is easy to spot in Jenkins. 
We usually name this the same as project (magma) since the template name already has the "go-clm" in the name:
"{project-name}-go-clm-{stream}"
This should translate to: "magma-go-clm-master"